src/client/clientobject.h

```diff
@@ -90,10 +90,10 @@ class ClientActiveObject : public ActiveObject
 	static std::unordered_map<u16, Factory> m_types;
 };
 
-struct DistanceSortedActiveObject
+class DistanceSortedActiveObject
```

Comment:
I've been wondering, does doing this introduce any overhead over `struct`s?

Reply:
I don't think so. AFAIK a struct is the same as a class, just that all access defaults to public.

Reply:
I could have left it a struct and just mark d private. Would you prefer that? It does not make a difference, though.

Reply:
No, it's fine.